docs/workflows/applications.md

```diff
@@ -0,0 +1,132 @@
+An application can also be created from the [Beyond Identity API](https://developer.beyondidentity.com/api/v1). Before making any API calls you'll want to generate an API access token. Check out [API Tokens](./api-token) for help creating an access token.
+
+### Create an Application Resource Server
+
+Before creating an application by API, you may want to create a resource server. A resource server is a namespace for application scopes. Application scopes is a set of all scopes supported by the application. When an application is created without a resource server, this application may provide authentication (identity) but not authorization (access). If your application doesn't provide multiple levels of access -- like admin access vs user access, then there might not be a need for a resource server.
+
+When creating an application with the [admin console](#create-application-from-admin-console), if the application lives in the admin realm, then the application is created with the management api resource server. Otherwise, it is currently created without a resource server.
+
+If you create a resource server, make note of the response `id`, as you'll need the resource server ID when creating an application.
+
+<MultiLanguageCodeBlock
+curl='curl "https://api-$(REGION).beyondidentity.com/v1/tenants/$(TENANT_ID)/realms/$(REALM_ID)/resource-servers" \
+-X POST \
+-H "Authorization: Bearer $(TOKEN)" \
+-H "Content-Type: application/json" \
+-d "{\"resource_server\":{\"display_name\":\"$(APPLICATION_DISPLAY_NAME)\",\"identifier\":\"$(APPLICATION_URI)\",\"scopes\":[\"$(SCOPE)\"]}}"'
+title="/resource-servers"
+/>
+
+### Create an Authenticator Config
+
+Before creating an application by API, you'll need to create an Authenticator Config by setting the application's Configuration Type, Invocation Type, Invoke URL and Trusted Origins. See configuring the [Authenticator Config](#authenticator-config) from the admin console for help setting the appropriate options.
+
+Make note of the response `id`, as you'll need the Authenticator Config ID when creating an application.
+
+<MultiLanguageCodeBlock
+curl='curl "https://api-$(REGION).beyondidentity.com/v1/tenants/$(TENANT_ID)/realms/$(REALM_ID)/authenticator-configs" \
+-X POST \
+-H "Authorization: Bearer $(TOKEN)" \
+-H "Content-Type: application/json" \
+-d "{\"authenticator_config\":{\"config\":{\"type\":\"$(CONFIGURATION_TYPE)\",\"invoke_url\":\"$(INVOKE_URL)\",\"trusted_origins\":[\"$(TRUSTED_ORIGIN)\"],\"invocation_type\":\"$(INVOCATION_TYPE)\"}}}"'
+title="/authenticator-configs"
+/>
+
+### Create an Application
+
+Once you have created an [Authenticator Config](#create-an-authenticator-config) and possibly a [Resouce Server](#create-an-application-resource-server), you can create an application.
+
+While the Resouce Server ID is optional, make sure to have the Authenticator Config ID for the applications request. Set the application's Display Name, Client Type, Token Configuration, and redirect URI.
```

Comment:
The authenticator config ID is optional as well. Maybe it's not important for this guide, but I just wanted to call it out. You can create an application without authenticator config if you only want to use the client credentials OAuth flow.

Reply:
What happens if you create an application without an authenticator config for the authorization code flow? Does it default to the web authenticator?

Reply:
I spoke with Tomas and it sounds like the request will fail in that case. 

Updated the docs to reflect an optional authenticator config.